File: rb/exo_08.rb
# frozen_string_literal: true

print "On compte à rebours à partir de quand ?\n> "
numb = gets.chomp.to_i

abort('Entrez un nombre supérieur à 0') if numb.zero?

while numb >= 0
  puts numb
  numb -= 1
end

File: c/exo_08.c
#include <stdio.h>
#include <stdlib.h>
#include <string.h>

int main(void) {
  int i;
  char numb[5] = {0};
  printf("On compte à rebours à partir de combien ?\n> ");
  fgets(numb, 5, stdin);
  numb[strcspn(numb, "\n")] = 0;
  for (i = atoi(numb); i >= 0; i--)
	  printf("%d\n", i);
}
